utils/plugin_docs.py

```diff
@@ -0,0 +1,59 @@
+
+import os
+from distutils.dir_util import copy_tree
+from jinja2 import Environment, FileSystemLoader
+
+
+# Directory in plugin to pull documentation from
+PLUGIN_DOCS_DIR = 'docs'
+
+
+def import_plugin_docs(caldera_root_dir, sphinx_root_dir):
+    print(f'Importing plugin documentation')
+    doc_paths = copy_plugin_docs(caldera_root_dir, sphinx_root_dir)
+    create_index(sphinx_root_dir, doc_paths)
+
+
+def copy_plugin_docs(caldera_root_dir, sphinx_root_dir):
+    """Copy docs from plugins to sphinx-docs/plugins/{plugin_name/
+
+    :param caldera_root_dir: Path to CALDERA base directory
+    :param sphinx_root_dir: Path to sphinx directory
+    :return: List of copied Markdown and reStructuredText files
+    """
```

Comment:
Style nit: I may lobby for this change formally via a linter config, but I much prefer google style docstrings to sphinx style: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods